packages/picker-button/package.json

```diff
@@ -65,7 +65,7 @@
         "@spectrum-web-components/shared": "1.3.0"
     },
     "devDependencies": {
-        "@spectrum-css/pickerbutton": "6.0.0-s2-foundations.16"
+        "@spectrum-css/pickerbutton": "6.1.1"
```

Comment:
@Rajdeepc After thinking about it some more, I think updating picker button and textfield in this same PR is blowing the scope up a bit. I would probably vote to remove those version updates.